File: C/salesCalc.c
#include <stdio.h>

int main() {
    int quantitySold;
    printf("Games sold by Catarina: ");
    scanf("%d", &quantitySold);

    double costPerGame = 19.90;
    double totalSales = quantitySold * costPerGame;
    double bonus = (quantitySold / 15.0) * 0.08 * totalSales;
    double salary = 0.5 * totalSales;
    double totalEarnings = salary + bonus;

    printf("%.2f\n%.2f\n%.2f\n", totalSales, bonus, totalEarnings);

    return 0;
}

File: Haskell/salescalc.hs
main :: IO ()
main = do
    putStr "Games sold by Catarina: "
    quantitySold <- readLn :: IO Int

    let costPerGame = 19.90
        totalSales = fromIntegral quantitySold * costPerGame
        bonus = (fromIntegral quantitySold / 15) * 0.08 * totalSales
        salary = 0.5 * totalSales
        totalEarnings = salary + bonus

    print totalSales
    print bonus
    print totalEarnings
